uitools/examples/qml_quick/src/demos/CalloutDemo.qml

```diff
@@ -16,6 +16,7 @@ import Esri.ArcGISRuntime 100.13
 import Esri.ArcGISRuntime.Toolkit 100.13
 
 DemoPage {
+    //! [Set up Callout QML]
```

Comment:
❌  I think we could reduce the scope of this snippet. Showing the `mapViewContents: Component` section is very app specific and could read confusing in the standalone doc. I'd just move the tag around MapView and its children instead.